dist/init/linux-systemd/README.md

```diff
@@ -1,84 +1,127 @@
-# systemd unit for caddy
+# systemd Service Unit for Caddy
 
 Please do not hesitate to ask on
 [caddyserver/support](https://gitter.im/caddyserver/support)
-if you have any questions.
-Feel free to prepend to your question the username of whoever touched the file most recently,
-for example `@wmark re systemd: …`.
+if you have any questions. Feel free to prepend to your question
+the username of whoever touched the file most recently, for example
+`@wmark re systemd: …`.
 
-The provided file is written for **systemd version 229** or later!
+The provided file should work with systemd version 219 or later. It might work with earlier versions.
+The easiest way to check your systemd version is to look at the version of the installed package
+(e.g. 'sudo yum info systemd' on RedHat/Fedora systems).
 
-## Quickstart
+## Instructions
 
-In the following sections, we will assume that you want to run caddy
-as user `www-data` and group `www-data`, with UID and GID 33.
-Adjust this to your liking according to the preferences of your Linux distribution!
+We will assume the following:
+
+* that you want to run caddy as user `www-data` and group `www-data`, with UID and GID 33
+* you are working from a non-root user account that can use 'sudo' to execute commands as root
+
+Adjust as necessary or according to your preferences.
+
+First, put the caddy binary in the system wide binary directory and give it
+appropriate ownership and permissions:
 
 ```bash
-groupadd -g 33 www-data
-useradd \
+sudo cp /path/to/caddy /usr/local/bin
+sudo chown root:root /usr/local/bin/caddy
+sudo chmod 755 /usr/local/bin/caddy
+```
+
+Give the caddy binary the ability to bind to privileged ports (e.g. 80, 443) as a non-root user:
+
+```bash
+sudo setcap 'cap_net_bind_service=+ep' /usr/local/bin/caddy
+```
+
+Set up the user, group, and directories that will be needed:
+
+```bash
+sudo groupadd -g 33 www-data
+sudo useradd \
   -g www-data --no-user-group \
   --home-dir /var/www --no-create-home \
   --shell /usr/sbin/nologin \
   --system --uid 33 www-data
 
-mkdir /etc/caddy
-chown -R root:www-data /etc/caddy
-mkdir /etc/ssl/caddy
-chown -R www-data:root /etc/ssl/caddy
-chmod 0770 /etc/ssl/caddy
+sudo mkdir /etc/caddy
+sudo chown -R root:www-data /etc/caddy
+sudo mkdir /etc/ssl/caddy
+sudo chown -R www-data:root /etc/ssl/caddy
+sudo chmod 0770 /etc/ssl/caddy
```

Comment:
Now that we use **systemd** we can:

```bash
cat >/etc/tmpfiles.d/caddy.conf <<"EOF"
d /etc/caddy 0755 root www-data
d /etc/ssl/caddy 0770 www-data root
EOF
```

… and ship the corresponding file with a *Caddy package*.